docs/containers-storage.conf.5.md

```diff
@@ -96,7 +96,7 @@ storage of content and can allow the kernel to use less memory when running
 containers.
 
 containers/storage supports three keys
-  * enable_partial_images="true" | "false"
+  * enable_partial_images="true" | "true"
```

Comment:
This looks wrong options is true or falsenot true or true.

Reply:
fixed